conode/README.md

```diff
@@ -1,56 +1,612 @@
+```
+
+Let's Encrypt certificates expire every 90 days, so you will need
+to restart your conode when the `fullchain.pem` file is refreshed.
+
+## Run your conode
+
+Once the setup is done with one of the two options, you can finally run your
+conode depending on the previously chosen option.
+
+### Option 1: :computer: Run with the command line
+
+Simply launch the following:
+
+```bash
+$ conode server
+```
+
+It is recommended to use more verbose logging and this can be done with the `-d`
+option. It is also possible to specify a different location than the default one
+for the private.toml file with the `-c` option. If you used Docker to set up your
+conode, then the private.toml file is not in the default location. You can then
+use:
+
+```bash
+$ conode -d 2 -c ~/conode_data/private.toml server
+```
+
+You can use the `-h` for a description of the available commands.
+
+If you want to run the server in the background, you can use the `screen`
+program:
+
+```bash
+screen -S conode -d -m conode server
+```
+
+To enter the screen, type `screen -r conode`. You can detach from it with
+`<ctrl-a> d`.
+
+
+### Option 2: :whale: Run with docker
+
+Type the following to start the conode program with docker:
+
+```bash
+$ docker run --restart always -d -p 7770-7771:7770-7771 --name conode -v ~/conode_data:/conode_data dedis/conode:latest
+```
+
+Because it will run detached, you can use `docker logs -f conode` to see the
+logs. It will be restarted on the next boot as well. To stop it, you can use
+`docker stop`. Then, you must remove it with `docker rm <id>` (the id is found
+with `docker ps -a`).
+
+Pro tip: you can use the following options for docker in order to gracefully
+handle the logs and prevent a disk saturation: 
+`--log-opt max-size=10m --log-opt max-file=4 --log-opt compress=true`
 
-## Creating Your own Cothority
+If you have systemd, you can simply copy the `conode.service` file and add it to
+your systemd-startup. Of course, you should do this as a non-root user:
 
-For most of the apps you need at least 3 running nodes. Once you have them up
+```
+$ wget https://raw.githubusercontent.com/dedis/cothority/conode/conode.service
+$ systemctl --user enable conode.service
+$ systemctl --user start conode
+```
+
+Unfortunately, systemd doesn't allow a user to run a service at system startup,
+and all user services get stopped once the user logs out!
+
+### Option 3: `run_nodes.sh`
+
+For development purposes, the `run_nodes.sh` script can be used to launch
+multiple conodes. For example, the following command:
+
+```bash
+$ ./run_nodes -d tmp -v 2 -n 5
+```
+
+will run 5 conodes and save their files in the tmp directory with verbosity of
+2. The file containing all the public configurations will be in
+"tmp/public.toml".
+
+# Operating a conode
+
+## Backups
+
+On Linux, the following files need to be backed up:
+1. `$HOME/.config/conode/private.toml`
+2. `$HOME/.local/share/conode/$PUBLIC_KEY.db`
```

Comment:
the db files are not hex-encoded public keys, they are the sha256 of the public keys (see the `dbFileName()` function in onet)

Reply:
probably just ask the user to backup the whole conode folder